### DISCUSSION TITLE
Live timing: Swap Last and Best time position

### DISCUSSION THREAD

**Discussion by meizenga** [f1multiviewer/issue-tracker#171] [Feature requests] [answered]
Suggest to swap the Best and Last time of a driver so that Last time is on top. Rationale: When watching live timing during the race and looking up a driver, I want to see his last time. Best time is not interesting during the race, want to track progress and see increasing/decreasing gaps.

<img width="320" alt="image" src="https://user-images.githubusercontent.com/23551316/187137720-f1a056bc-e409-4ecf-9167-c8b12acba4ac.png">

Additionally the Best time could get a different color to distinguish Best and Last even better.

**Comment by stefanverkerknl** [comment]
I agree with this. I've had multiple times when I looked at the best laptime instead of the last laptime and I was confused. I find the difference between them too small to quickly see which time I have to look at. It's too close to each other without any difference in size/color/font/etc.

**Reply by dseven** [reply]
> I agree with this. I've had multiple times when I looked at the best laptime instead of the last laptime and I was confused.

Same - intuitively I expect the top time to be the most recent one, at least during the race. Quali might be a bit different, though - there you primarily care about the best time.....

**Comment by f1multiviewer** [comment, marked as answer]
This has been solved in the latest release 💚
